rfcs/rfc-005-branches-tags-and-channels.md

```diff
@@ -0,0 +1,52 @@
+```
+========================= master branch (edge channel) =======================>
+         \                      \                     \
+          \___v0.7.0 tag         \                     \
+           \                      \         v0.9.0 tag__\
+            \          v0.8.0 tag__\                     \
+ v0.7.1 tag__\                      \                 v0.9 branch (beta channel)
+              \___v0.7.2 tag         \___v0.8.1 tag
+               \                      \
+                \                      \
+           v0.7 branch         v0.8 branch (stable channel)
+
+```
+
+## Branches and Tags
+
+### master branch
+All new development occurs on the `master` branch.
+
+Bug fixes that affect a `vX.Y` branch are first made on `master`.  Once the bug
+fix lands on `master` it is cherry-picked into the `vX.Y` branch and potentially
+the `vX.Y+1` branch.  The exception to this rule is when a bug fix for `vX.Y`
+doesn't apply to `master` or `vX.Y+1`.
+
+Immediately after a new stabilization branch is forged, the minor version of the
```

Comment:
This would be easier to grok if a little more concrete. "the minor version in Cargo.toml on the `master` branch." And should we add "the patch version in Cargo.toml on the new branch"?  If we always tag *before* branching (suggesting branching is done on demand), that'd be the case.

Reply:
I'll mention Cargo.toml, good point.   

We don't tag before branching in this scheme.  We branch from master to vX.Y, and then tag vX.Y.Z from the vX.Y branch.  Major/minor version bumps happen on master, patch version bumps happen on vX.Y